### PR TITLE
Ft update airflow cicd

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Adds custom airflow helm chart
- Updates airflow's cicd pipeline

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- n/a

**_HOW DO I TEST OUT THIS PR?_**
- Do a code analysis

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- n/a

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- n/a

### CODE REVIEW

File: k8s/airflow/templates/service.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  namespace: {{ .Values.app.namespace }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: {{ .Values.service.targetPort }}
+      protocol: {{ .Values.service.protocol }}
+      nodePort: {{ .Values.service.nodePort }}
+  selector:
+    app: {{ .Values.app.label }}
```
Comment (123MwanjeMike) [inline k8s/airflow/templates/service.yaml:15]: [airflow service](https://github.com/airqo-platform/AirQo-api/pull/1297/commits/7972d5be550ce209d9ade09654c9e7d1c9855d92)

File: k8s/airflow/templates/hpa.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  metrics:
+    {{- if .Values.autoscaling.targetCPUUtilizationPercentage }}
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: {{ .Values.autoscaling.targetCPUUtilizationPercentage }}
+    {{- end }}
+    {{- if .Values.autoscaling.targetMemoryUtilizationPercentage }}
```
Comment (123MwanjeMike) [inline k8s/airflow/templates/hpa.yaml:24]: [airflow service](https://github.com/airqo-platform/AirQo-api/pull/1297/commits/7972d5be550ce209d9ade09654c9e7d1c9855d92)

File: k8s/airflow/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+      maxUnavailable: 1
+  minReadySeconds: 5
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        app: {{ .Values.app.label }}
```
Comment (123MwanjeMike) [inline k8s/airflow/templates/deployment.yaml:27]: [airflow deployment template](https://github.com/airqo-platform/AirQo-api/pull/1297/commits/9bab54eb6419b1c3bf535974a4552023ada07a0b)

File: k8s/airflow/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+appVersion: "1.0.0"
+description: AirQo custom Airflow Helm Chart
+name: airflow
+home: https://airqo.net
+version: 0.1.0
+maintainers:
+  - name: AirQo
+    email: support@airqo.net
+    url: https://airqo.net
```
Comment (123MwanjeMike) [inline k8s/airflow/Chart.yaml:10]: [airflow full helm chart](https://github.com/airqo-platform/AirQo-api/pull/1297/commits/b1c1282ce9f6372ed22a70dbe1a337ed571689d7)

File: .github/workflows/deploy-apis-to-staging.yml [hunk 5]
```diff
@@ -564,17 +569,18 @@ jobs:
       - name: Setup Cloud SDK
         uses: google-github-actions/setup-gcloud@v0
 
+      - name: Login to K8S
+        uses: azure/k8s-set-context@v2
+        with:
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-staging.yml:574]: [update airflow staging cicd pipeline](https://github.com/airqo-platform/AirQo-api/pull/1297/commits/aba994d41f9866d05b3b72f9ec1a761c295b2881)

File: .github/workflows/deploy-apis-to-production.yml [hunk 6]
```diff
@@ -564,17 +569,18 @@ jobs:
       - name: Setup Cloud SDK
         uses: google-github-actions/setup-gcloud@v0
 
+      - name: Login to K8S
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-production.yml:572]: [update airflow production cicd pipeline](https://github.com/airqo-platform/AirQo-api/pull/1297/commits/8baddf33cafcafbba542515eedbfda215ec5d527)

**Review by 123MwanjeMike — COMMENTED** [review]
Guiding comments

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**